portfolio/dakeys.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "despindola-sps-spring21",
+  "private_key_id": "2ab10a411508878f11101d7bcacc64fda2e1c6ce",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCcJyUic/lNHsJy\nd+Lr9YOGijU7rMN4peveDTtyRL0kNW4aZ47em++tJ1x8XpQgf04HUhF9f07m34BH\nclxp3sZW/Q+W6vXdBWJGUehlgnqikl3ISnD86XSKJwi4gt4iCyd+otqWpXE3dm/E\nDepDi8k2vAXtd1yKkBc5DZUPRWL2Q+orz+gT4ZGeXPy7HgQeyhs/AbqMwG1ipgSc\nfvE0zT56tyvG0e+0AQl0TNfLVvHeZVlK6C3YaO51+VhiDo9qlQsc+2B9rT+mQvIG\nG0FhovYnwUa3B6QTaC40A1fK8yVT8sx/sMOuhYwBdjj2noDnvVsDOHqbur1ra4Zk\nk8Fx8HfDAgMBAAECggEAOScutUyDW9G9U85DdR+4Ux702vGh3d/E09I+BtXX8I/J\ny8pJhXrHn7W/aRno13gg5BcKpcdZ5rC5L+1Qm8NZDu+yrAc1k/7reFd0WusF3cj6\noFt2vWrpJ8OfsoDbstQoUXZj0S2eobEsuTJqouxf6jOpa7/7VA/nIA+zlhJMZVvl\n1qEsm1s8zJCMja/gzQxX1TlwjC4tHPYSSyZATzGOE9k9/rA2NpkfM/cS+SaC/ARq\nA8awK/rr7CHyaNXOPOxPrhs4ZFHb/9aXJ4NVLaVxzNwnuN4DY0LNlT+rue1+bUGW\n0w9ZbU/P4oFBuEYWoIdGtR2XCF4EJYv3D/o5acfvuQKBgQDIXM6f9vw5jtsLfvsc\nlbUSR9tXGMyk2qLr7JWzupVUguTMZzw+9uVOxb2QHi11bAX0wad4PU+TvNxHHH1m\nF4WfMsP07FEmnF9BKg3mw9DQSPlQ4wZM7Bq/24S41n1OdY3okWfvXHgIYNdKr1Mm\nyKsB7jPyhpSmg6bEn0yZ9nIK7QKBgQDHg5qhrlox6J/H/nodTdZ+6UiV36ahCrG6\nGwZOl13G4ic91KNPOgEf+9t16DUg2aVJRtSkCCjnuhYMxTJQibU+XEYkkaUt6A/o\nKcDITuF9eiPwm+yiF6/HgLSn4aG7OBXOgRfDT+/lYjeu4OTt+0b+HdyrHCu1V629\nmSpovo5HbwKBgD394PxDWbShS4qaBi3I8BYw319qT97tupzOPV83an6UZSANDOna\nUAZ4TfdqLtx2yLz6scsVBSsLswfZe777ph8kgNaly1Hr7djmnxh4G4JmIaVyKH/i\nSUB1NfJOgsSkwHxliMfZT1YgSFxWNJ76VVaqka4HD7UZZJ7PPs8gHhbdAoGADpaY\n3+mXvr57sGvzEeF3JaB5JjxMOspFZI5FT8nRmZE/vU2nA0wCeQw5Z82q1NQVOd5r\n/fJ6yzbYFeb1Xt4y2nR0zIe5xDmLsDYdWogqBLrBlf5Qtyl0OOo4S248HSxkq8zU\np/tYVzVvTOI02xHchwg38msE5yWaFpr0duQ4ExECgYAD1YJeefip29sTZAjH4wzg\nnKzaGSKSDIUpKIX86RIYglfqivanCtxKHpinfgWM4D8lhATZ3t5aSuBhRyzi5QcA\nORzo/KCpj9q46iAfTZI4nVbd+5sIe0uP2UOf5TkfBzw+SyI/CHTjopFENyiv2HWl\nuTSiPC7ENponkpAmYJw+mw==\n-----END PRIVATE KEY-----\n",
```

Comment:
This file shouldn't be a part of the commit, because it allows anyone to use your account's API quota for Google Cloud.